util/db/migrations/20210920144047-sfo-fix-any-broken-child-records-with-deleted-parents.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+if (process.env.NODE_ENV === 'production') {
+  module.exports = {
+    up: async (queryInterface, Sequelize) => {
+      await queryInterface.sequelize.query(
+        `
+        UPDATE sfo."Setts" SET "deletedAt" = CURRENT_TIMESTAMP WHERE "deletedAt" IS NULL AND "ApplicationId" IN (SELECT id FROM sfo."Applications" WHERE "deletedAt" IS NOT NULL)`,
+        {
+          type: Sequelize.QueryTypes.UPDATE
+        }
+      );
+
+      await queryInterface.sequelize.query(
+        `
+        UPDATE sfo."Returns" SET "deletedAt" = CURRENT_TIMESTAMP WHERE "deletedAt" IS NULL AND "ApplicationId" IN (SELECT id FROM sfo."Applications" WHERE "deletedAt" IS NOT NULL)`,
+        {
+          type: Sequelize.QueryTypes.UPDATE
+        }
+      );
+    },
+
+    down: () => {
```

Comment:
I think this should be...
```diff
-    down: () => {
+    down: async () => {
```
...since you're returning a Promise from this function. It's _unlikely_ since we don't ever really rewind our migrations but this could cause a race condition if we ever had to.

FYI, if you just wanted to quieten the linting error, you could have also prefixed the arg names with `_`

```diff
-    down: async (queryInterface, Sequelize) => {
+    down: async (_queryInterface, _Sequelize) => {
```